functions/completed-checks/v1.js

```diff
@@ -33,27 +33,27 @@ async function handleCompletedCheck (context, completedCheckMessage) {
   try {
     checkData = await sqlUtil.sqlFindCheckWithFormDataByCheckCode(completedCheckMessage.checkCode)
   } catch (error) {
-    context.log.error(`completed-check: ERROR: failed to retrieve checkData for checkCode: ${completedCheckMessage.checkCode}`)
+    context.log.error(error)
```

Comment:
just the error message please.  

does the entire error get logged when the function receives the thrown error?